ensembl_metadata/models/genome.py

```diff
@@ -0,0 +1,160 @@
+from django.db import models
+from ensembl_metadata.models.assembly import Assembly
+from ensembl_metadata.models.datarelease import DataRelease
+from ensembl_metadata.models.organism import Organism
+import uuid
+
+
+class Division(models.Model):
+    division_id = models.AutoField(primary_key=True)
+    name = models.CharField(unique=True, max_length=32)
+    short_name = models.CharField(unique=True, max_length=8)
+
+
+class Genome(models.Model):
+    genome_id = models.IntegerField(primary_key=True)
+    genome_uuid = models.CharField(max_length=128, default=uuid.uuid1, unique=True)
+    assembly = models.ForeignKey(Assembly, models.CASCADE, related_name='genomes')
+    organism = models.ForeignKey(Organism, models.CASCADE, related_name='genomes')
+    genebuild = models.CharField(max_length=255)
+    has_pan_compara = models.IntegerField()
+    has_variation = models.IntegerField()
+    has_microarray = models.IntegerField()
+    has_peptide_compara = models.IntegerField()
+    has_genome_alignments = models.IntegerField()
+    has_synteny = models.IntegerField()
+    has_other_alignments = models.IntegerField()
```

Comment:
All these could be turned into https://docs.djangoproject.com/en/3.1/ref/models/fields/#booleanfield

Reply:
Basically that doesn't change the field, but the controls applied when using it in Python. More over you can simply assign Boolean values in code.